app/services/exports/trainee_search_data.rb

```diff
@@ -205,6 +206,12 @@ def course_allocation_subject(subject)
       SubjectSpecialism.find_by(name: subject)&.allocation_subject&.name
     end
 
+    def course_education_phase(trainee)
+      return "Early years" if trainee.early_years_route?
```

Comment:
Same as above with early years. This could just return the string with first char upcased